udata_front/settings.py

```diff
@@ -78,5 +78,8 @@
 # Metrics API
 METRICS_API = None
 
+# recherche-entreprises.api.gouv.fr
+SEARCH_SIREN_URL = 'https://recherche-entreprises.api.gouv.fr/search'
```

Comment:
@maudetes do we want to use `None` as default to avoid requests from other udata forks ? Also, following https://github.com/datagouv/publier.etalab.studio/issues/42, we should add a `mtm_campaign` to our request to improve the API analytics

Reply:
Yes and yes! :pray:

Reply:
We should still set this var as `None` in default settings making it optional